app/design/frontend/Magento/blank/Magento_Catalog/web/css/source/_module.less

```diff
@@ -115,15 +115,14 @@
     }
 
     .product-image-photo {
+        position: absolute;
+        top: 0;
+        right: 0;
         bottom: 0;
-        display: block;
-        height: auto;
         left: 0;
+        display: block;
+        width: auto;
         margin: auto;
-        max-width: 100%;
-        position: absolute;
-        right: 0;
-        top: 0;
     }
 
```

Comment:
you should sort styles properties for both file less changed

Reply:
I did sort it for both files, what did I mis?

Reply:
magento require all properties in less should sort alphabetical. You can see fail static tests report for more details

Reply:
okay fixed it.